src/index.js

```diff
@@ -0,0 +1,151 @@
+import './style.css';
+
+const getDataFromLocalStorage = () => {
+  const todoList = localStorage.getItem('todoList');
+  return JSON.parse(todoList);
+};
+
+const updateView = () => {
+  const todoList = getDataFromLocalStorage();
+  const todo = document.getElementById('todo-list');
+  todo.innerHTML = '';
+  if (todoList) {
+    todoList.forEach((todoListItem) => {
+      todo.innerHTML += `
+        <li><hr></li>
+        <li class="todo ${todoListItem.editable ? 'active' : ''}" >
+            <div>
+              <input type="checkbox" name="checkbox" id="checkbox_${todoListItem.id}" ${todoListItem.completed ? 'checked' : ''}>
+              <input id="input_${todoListItem.id}" ${todoListItem.editable ? '' : 'disabled="true"'} value="${todoListItem.description}" class="borderless ${todoListItem.completed ? 'edit' : ''} " />
+            </div>
+            <section class="action">
+              <button type="button" ${todoListItem.editable ? 'hidden' : ''} id="toggleMode_${todoListItem.id}" class="fa fa-pencil"></button>
+              <button type="button" ${todoListItem.editable ? '' : 'hidden'} id="save_${todoListItem.id}" class="fa fa-check"></button>
+              <button  type="button" ${todoListItem.editable ? '' : 'hidden'} id="delete_${todoListItem.id}" class="fa fa-close"></button>
+            </section>
+        </li>
+      `;
+    });
+  }
+};
+
+const storeItem = (items) => {
+  if (items.length > 0) {
+    localStorage.setItem('todoList', JSON.stringify(items));
+  } else {
+    localStorage.clear();
+  }
+};
+
+const clearLocalStorage = () => {
+  const todoListArr = getDataFromLocalStorage();
+  if (todoListArr) {
+    let counter = todoListArr.length;
+    while (counter > 0) {
+      if (todoListArr[counter - 1].completed) {
+        todoListArr.splice(counter - 1, 1);
+      }
+      counter -= 1;
+    }
+    storeItem(todoListArr);
+    updateView();
+  }
+};
+
+const clearInput = () => {
+  document.getElementById('todoListInput').value = '';
+};
```

Comment:
This function is called once and its already 1 line. No need to make it a seperate function.